src/test/analyzer_test.ts

```diff
@@ -168,6 +168,21 @@ suite('Analyzer', () => {
           ['polymer.html', 'foo.js', 'foo.css']);
     });
 
+    test('polymer css import scanner', async() => {
+      let contents = `<html><head>
+          <link rel="import" type="css" href="bar.css">
```

Comment:
I think it's worth bringing up a little example element locally to try to figure out how Polymer handles these links. (I know that they're widely used by some of our very important customers, or I might not push on this as much, as it **is** deprecated).

As for `<template>` contents, I think the goal should be to try hard to do the Correct thing in Analyzer. I'm confident that we're going to end up treating Polymer template contents (i.e. template contents inside `<dom-module>`s and `<template is="dom-bind">` specially).